iam/snippets/snippets.go

```diff
@@ -0,0 +1,232 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package snippets
```

Comment:
We've started writing each sample in separate files so the region tag can include the import statements. Might want to proactively do it here so we don't have to update later. For example, see https://github.com/GoogleCloudPlatform/golang-samples/pull/706.

Reply:
This pattern deviates from the rest of snippet in the docs. For example, see: https://cloud.google.com/iam/docs/creating-managing-service-accounts#iam-service-accounts-create-csharp
Is there a reason why you're moving towards this approach?

Reply:
We want samples to be idiomatic for every language. That differs a bit for each language when it comes to regions inside/outside the function declaration, imports inside or outside (or can they be directly in the function), etc. Including the imports in region seems the most clear for Go.

Not a requirement for now.